pkg/asset/internal/templates.go

```diff
@@ -51,6 +49,7 @@ spec:
         - --kubeconfig=/etc/kubernetes/kubeconfig
         - --require-kubeconfig
         - --lock-file=/var/run/lock/kubelet.lock
+        - --containerized
```

Comment:
Doesn't this break `mount` binaries that are present in hyperkube but not on container linux?